src/Jackalope/ObjectManager.php

```diff
@@ -654,6 +654,26 @@ public function getBinaryStream($path)
         return $this->transport->getBinaryStream($this->getFetchPath($path, 'Node'));
     }
 
+    public function hasNodeType($nodeTypeName)
+    {
+        return $this->transport->hasNodeType($nodeTypeName);
+
+        try {
```

Comment:
yeah .. I mainly still left it in .. since we actually have to think a bit more about the transport API since I have not prototyped what changes are needed in the Jackrabbit transport for this.